src/components/Map/index.js

```diff
@@ -21,8 +22,19 @@ const reactLeafletMapStyle = {
   height: 'auto',
 };
 
-const Map = ({ neighborhoods, onSelect }) => (
-  <div style={reactLeafletMapStyle}>
+const activePathOptions = {
+  fill: false,
+  stroke: true,
+  color: '#ee495c',
+  weight: 3,
+};
+
+const Map = ({ neighborhoods, onSelect, activeNeighborhood }) => {
+  const activeDatum = neighborhoods && activeNeighborhood && neighborhoods.features.find(
+    neighborhood => neighborhood.id === activeNeighborhood,
+  );
```

Comment:
This would probably be better in a selector so it doesn't run every render . . . but also we probably need a cleaner system for tracking the active neighborhood. I say just get it in, we need to get this done!